pkpdapp/pkpdapp/urls.py

```diff
@@ -27,4 +27,12 @@
     path('generic/', views.GenericView.as_view(), name='generic'),
     path('simulate/', include('simulate.urls')),
     path('django_plotly_dash/', include('django_plotly_dash.urls')),
+    path('dataset/<int:pk>/',
+         views.DatasetDetailView.as_view(), name='dataset-detail'),
+    path('pkpd_model/<int:pk>/',
```

Comment:
Can we call the URL just `model`? 
1. Underscores in URLs don't seem to occur that often
2. Users don't won't have to deal with database models, so there isn't really potential for confusion
3. I think in the long run it makes sense to go away from the concept of a PKPD model, which really is a PK, PD and disease model (or any of these submodels).

Reply:
well, the reason its `pkpd_model` is because that is the name of the view, which in turn is based on the name of the model. We'll need to have some sort of naming convention, so this seems to be the most convenient? We'll also need some way of handling "spaces" in url names. Wether that is `_` or `-`, or other I don't really mind.

Reply:
Ok sounds good! I guess I was already thinking about relaxing pkpd model to be any (PK, PD, DB) model.